code/modules/industrial_lift/industrial_lift.dm

```diff
@@ -389,7 +389,7 @@ GLOBAL_LIST_EMPTY(lifts)
 					continue
 				to_chat(collided, span_userdanger("[src] collides into you!"))
 				playsound(src, 'sound/effects/splat.ogg', 50, TRUE)
-				var/damage = rand(5, 10) * collision_lethality
+				var/damage = rand(9, 28) * collision_lethality
```

Comment:
doesn't this make the minimum damage less than before?

Reply:
no

Reply:
unless you mean before as in the pre review version of this pr, in which case yes